actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add SameSite to cookies.
+
+    `SameSite` cookie is an addition to the CSRF protection.
+    For now, the default is `:no_protection` which makes it backward compatible.
+    By switching from `:no_protection` to `:lax` in `config/initializers/cookies.rb`,
+    cookies won't be sent by browsers in cross-site POST requests.
+    `:strict` will disable cookies from being sent in both cross-site GET and POST requests.
+
+    More info [here](https://tools.ietf.org/html/draft-west-first-party-cookies-07)
+
+    *Cédric Fabianski*
+
```

Comment:
Setting `same_site: …` is already possible with cookies since it's supported in Rack. Let's clarify that this makes it a default setting that applies to all cookies.